### PR TITLE
Add GitPod config files

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (JohnNiang) [inline .gitpod.Dockerfile:1]: Useless blank line.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: .gitpod.Dockerfile (hunk 1 above)
Comment (LinuxSuRen) [reply .gitpod.Dockerfile]: Done.

**Pull request merged**